class.jetpack.php

```diff
@@ -568,28 +576,36 @@ static function delete_active_modules() {
 	public function add_configure_hook() {
 		global $wp_filter;
 
-		$current_priority = has_filter( 'plugins_loaded', array( $this, 'configure' ) );
-		if ( false !== $current_priority ) {
-			remove_action( 'plugins_loaded', array( $this, 'configure' ), $current_priority );
+		// If there is no handler set for the plugins_loaded hook yet, set it and return.
+		if ( ! isset( $wp_filter['plugins_loaded'] ) ) {
+			add_action( 'plugins_loaded', array( $this, 'configure' ), 0 );
```

Comment:
Should we set the `$this->highest_priority` value to 0 here before returning?

Reply:
You are right @kbrown9. I would even go a step further by setting it as the default value of the private property. And this would also address your [second comment](https://github.com/Automattic/jetpack/pull/14807/files#r385211681).

Reply:
Fixed that, thank you!